index.d.ts

```diff
@@ -1,6 +1,10 @@
+/// <reference types="node" />
+/// <reference types="redis" />
+
+import { EventEmitter } from 'events';
 import { ClientOpts } from 'redis';
 
-declare class BeeQueue {
+declare class BeeQueue<T> extends EventEmitter {
```

Comment:
@bondz sorry, for some reason I missed your PR and my mention.

I have briefly skimmed your PR and I really appreciate it! However I would suggest two changes: 

```suggestion
declare class BeeQueue<T = unknown> extends EventEmitter {
```

This will fix the following issue:
```typescript
import bq from "bee-queue"
class Foo {
	private _queue: bq; // This will result in error "Generic type 'BeeQueue<T>' requires 1 type argument(s)."
}
```

Reply:
Yes, I see what you mean. Don't you think `any` would be easier for users, same as being untyped. `unknown` is best, I know but users would have to explicitly cast or validate the data type everywhere.

Reply:
Yes you are correct! I did not mention this issue, because I casted the type to the correct one😅
any would be better for backwards compatibility. I will edit my suggestion